### PR TITLE
66 dashboard optimize bootstrap

### PR DESCRIPTION
**Pull request by Starbeuck**

### CODE REVIEW

File: docs/developers/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Dashboard documentation
+
+The dashboard is written with vanilla JS, CSS and HTML (and love). We made the choice to not used any frameworks like Angular or Vue.js... because dashboard is a very small web site.
+
+This dashboard uses only two externals frameworks, which are Bootstrap and Socket.io.
+
+Socket.io is useful to dynamically received events from Copilot and displayed it.
```
Comment (gdoffe) [inline docs/developers/dashboard.md:7]: s/received/receive/
s/displayed it/display them/

File: docs/developers/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Bootstrap is a framework very powerful which comes with layout conception (grid to help us with responsively) and components (like modal that we used). But Bootstrap is a very large and heavy framework and we only use a percentage of what it offers. So I made the choice to purge it to reduce loading time on dashboard.
```
Comment (gdoffe) [inline docs/developers/dashboard.md:9]: s/used/use/

**Review by gdoffe — APPROVED** [review]
Good job (from my noobie web developper point of view :P).

Few typos, I already approve, change them and merge !

File: docs/developers/dashboard.md (hunk 2 above)
Comment (Starbeuck) [reply docs/developers/dashboard.md]: Done

File: docs/developers/dashboard.md (hunk 1 above)
Comment (Starbeuck) [reply docs/developers/dashboard.md]: Done

**Review by ecourtois — APPROVED** [review]

**Pull request merged**